pkg/services/thumbs/crawler.go

```diff
@@ -216,7 +220,7 @@ func (r *simpleCrawler) shouldWalk() bool {
 	return r.status.State == running
 }
 
-func (r *simpleCrawler) walk(ctx context.Context) {
+func (r *simpleCrawler) walk(id int, ctx context.Context) {
```

Comment:
context should be first argument (just a convention in Go)

Reply:
fixed! :nail_care: